app/index.js

```diff
@@ -1,11 +1,13 @@
 require('./insights').setup()
 require('log-timestamp')
 const { createServer } = require('./server')
+const { createAzuriteInfrastructure } = require('./blob-storage')
 
 const init = async () => {
   const server = await createServer()
   await server.start()
   console.log('Server running on %s', server.info.uri)
+  await createAzuriteInfrastructure()
```

Comment:
We only use Azurite if we're running locally so we should put conditional logic around this.

Consider if any of the functionality is needed when connecting in SND4 eg do we need to check the blob storage is available? This should result in a different log message as currently the service logs `Azurite infrastructure created successfully` when in SND4 which does not use Azurite.

Reply:
I think we should rename this function as we're not creating any Azurite infrastructure when in deployed environment so its a bit misleading. Perhaps `connectToStorageAccount` or similar?